.github/workflows/main.yml

```diff
@@ -33,5 +33,6 @@ jobs:
           git add gas_info.json
           git add history/gas_info_history.csv
           git add history/gas_info_history.json
+          git add history/img/plot.png
```

Comment:
@Dntfreitas This line should be removed since you are not writing the file on the repo any more.